src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        I am
+        {person.age}
+      </p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.partnerName}
+        is my
+        {person.sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
here is you should change only the text inside `<p>` tag.

1. move condition into a variable `const partnerStatus = sex === 'm' ? 'wife' : 'husband'`
2. use like this. Example:

```suggestion
      <p className="Person__partner">{isMarried ? `${partnerName} is my ${partnerStatus}` : 'I am not married'</p>
```

Reply:
<p className="Person__partner">{isMarried ? `${partnerName} is my ${partnerStatus}` : 'I am not married'</p>
this is not working
Parsing error: Unterminated regular expression
may be so
<p className="Person__partner">{person.isMarried ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'}</p>

Reply:
yes. since as i asked at previous comment you need to use object destructing first.
Then create a variable `const partnerStatus = sex === 'm' ? 'wife' : 'husband'`

Reply:
may be destructing is a good practice to use object, but the condition of this task is not require this. isn't it? if i do not use destructing is it a mistake?

Reply:
no, it's not a mistake. It's a good practice that developers apply at real projects. During your studying you'll have requirements yet mentors don't only check requirements. They're here to help to achieve something more. Especially with improving your solutions